examples/nextjs/starter/snapwp.config.ts

```diff
@@ -1,5 +1,8 @@
 import type { SnapWPConfig } from '@snapwp/core/config';
+import { ApolloQueryClientAdapter } from '@snapwp/apollo';
```

Comment:
I see no reason to hold on to the 'Adapter' semantics here. It can be just name 'ApolloQueryEngine'